packages/darkice/PKGBUILD

```diff
@@ -0,0 +1,54 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot de>
+# Contributor: Sam Mulvey <archlinux at sammulvey.com>
+# Contributor: Andrea Scarpino <andrea@archlinux.org>
+# Contributor: Ganjolinux aka Basalari David <ganjolinux@gmail.com>
+# Contributor: speps <speps at aur dot archlinux dot org>
+# Contributor: B3l3tte <ouack23 at yahoo.fr>
+
+pkgname=darkice
+pkgver=1.4
+pkgrel=3
+pkgdesc='Reads live audio from backends, encodes it and streams it to a server'
+arch=(aarch64 armv6h armv7h x86_64)
+url='http://www.darkice.org/'
+license=(GPL3)
+depends=(gcc-libs)
+makedepends=(alsa-lib faac flac gcc-libs jack lame libpulse libsamplerate libvorbis twolame opus)
```

Comment:
Actually, it should be the other way round. `gcc-libs` is a run-time dependency. Fixed in 1008f92. `namcap` will report it as redundant, because via the soname dependencies other packages are added as dependencies, which also depend on `gcc-libs`. But since this package is linking to it directly (as all C++ application will do) and it would depend on it, even if it had no other dependencies, it should be listed.

Reply:
`depends=(gcc-libs)` implies `makedepends=(gcc-libs)`, hence we should remove it here, or am I missing anything?

Reply:
Ah sorry, you actually changed that. Nvm then 👍